### PR TITLE
add macos mitigation for kleen-ritecorp.com

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:**

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Comment by shakyShane** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1230** <a href="https://app.graphite.dev/github/pr/duckduckgo/privacy-configuration/1230" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/privacy-configuration/1230?utm_source=stack-comment).

**Comment by shakyShane** [comment]
I checked the diff locally - on iOS a bunch of trackers on the allowlist just moved around

File: overrides/ios-override.json [hunk 1]
```diff
@@ -104,6 +104,24 @@
...
+            "settings": {
+                "allowlistedTrackers": {
+                    "monetate.net": {
+                        "rules": [
+                            {
+                                "rule": "monetate.net",
+                                "domains": [
+                                    "kleen-ritecorp.com"
+                                ],
+                                "reason": "blank page, and product carousels not working - https://app.asana.com/0/1200277586140538/1205303551800514/f"
```
Comment (icodebyamanda) [inline overrides/ios-override.json:118]: text descriptions are now deprecapted we usually add an issue link with the description of the issue. I made one here https://github.com/duckduckgo/privacy-configuration/issues/1231

File: overrides/macos-override.json [hunk 2]
```diff
@@ -510,6 +510,17 @@
...
                         ]
+                    },
+                    "monetate.net": {
+                        "rules": [
+                            {
+                                "rule": "monetate.net",
+                                "domains": [
+                                    "kleen-ritecorp.com"
+                                ],
+                                "reason": "blank page, and product carousels not working - https://app.asana.com/0/1200277586140538/1205303551800514/f"
```
Comment (icodebyamanda) [inline overrides/macos-override.json:521]: Same as above, here's the link to the issue description: https://github.com/duckduckgo/privacy-configuration/issues/1231

**Review by icodebyamanda — APPROVED** [review]

**Review by SlayterDev — APPROVED** [review]

**Pull request merged**